template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hue/HueConfigProviderTest.java

```diff
@@ -18,25 +18,26 @@
 import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
 import com.cloudera.api.swagger.model.ApiClusterTemplateVariable;
 import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessor;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.HueConfigProvider;
```

Comment:
I guess `HueConfigProvider` should be moved to the subpackage, too.

Reply:
correct, I forget it (i was on the train...)